index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Flixster</title>
+</head>
+<body>
+    
+    <div id="header">
```

Comment:
Consider using more [semantic markup](https://developer.mozilla.org/en-US/docs/Glossary/Semantics). This is a good example:

```suggestion
    <header>
```
Then in style.css:
```
header {
  display: block;
  ...
}
```